lib/v2/comicat/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/issue/:user/:repo?': ['Cyang39'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/21a30a5cf3c461ea0953085994c55ca8f8bfa83f/lib/v2/comicat/router.js#L2
```suggestion
    '/search/:keyword': ['Cyang39'],
```